pom.xml

```diff
@@ -56,15 +56,6 @@
         <module>tests</module>
       </modules>
     </profile>
-    <profile>
```

Comment:
This is not correct. We have an issue with the current build that it does not build and test `http-generator-helidon`. I'm fixing that.  After that we should find the EA JDK 22 build failing ... and then should be able to fix that with a bump to the parent pom.